genson/src/main/java/com/owlike/genson/Genson.java

```diff
@@ -609,6 +615,10 @@ public RuntimePropertyFilter runtimePropertyFilter() {
     return runtimePropertyFilter;
   }
 
+  public UnknownPropertyHandler unknownPropertyHandler() {
```

Comment:
Worth updating the [user doc](https://github.com/owlike/genson/blob/master/website/Documentation/Configuration.md) with this new c onfig option. Also some javadoc here would be nice.

Reply:
Agree on both points, but I was waiting for us to agree on what it is that should be documented ;-)

I think we are getting closer, I can add some docs.

Reply:
I'd document the simpler public API. So basically how to register an UnknownPropertyHandler and what it is. What are the existing defaults and how to use them. I wouldn't go in the details on how to implement one.